src/jsx/PostTweetContainer.jsx

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from "react";
+import { postTweet } from "../utils.js";
+
+class PostTweetContainer extends Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            postStatus: '',
+            postCharCount: 0,
+            postMessage: '',
+        };
+
+        this.updatePostCharCount = this.updatePostCharCount.bind(this);
+        this.postTweet = this.postTweet.bind(this);
+    }
+
+    updatePostCharCount(event) {
+        this.setState({ postMessage: event.target.value });
```

Comment:
you can set the state for multiple values.  Have two set states will trigger 2 renders